cs-amazon/src/main/java/io/cloudslang/content/amazon/entities/aws/Attribute.java

```diff
@@ -29,7 +29,7 @@
         this.value = value;
     }
 
-    public String getValue() {
+    private String getValue() {
```

Comment:
then for Architecture enum you should rename the getValue method to getArchitecture cause it does the same as you explained above(returns the same "Not relevant" etc.)